Gemfile

```diff
@@ -18,9 +18,7 @@ group :acceptance_testing do
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.0')
 end
 
-if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-  gem "scooter", *location_for(ENV['SCOOTER_VERSION'] || '~> 3.0')
-end
+gem "scooter", *location_for(ENV['SCOOTER_VERSION'] || '~> 3.0')
```

Comment:
We could do that; you can still run the specs locally, you just need to have an export for internal rubygems.  It's probably fine as is.